projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts

```diff
@@ -27,6 +36,12 @@ export function checkElement(tabElement: TabElement) {
       cy.focused().should('contain', tabElement.value);
       break;
     }
+    case TabbingOrderTypes.CHECKBOX_WITH_LABEL: {
+      cy.focused()
```

Comment:
I'm thinking about optimizing this with `.focused().within(...)` instead of searching the whole parent